com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -62,21 +67,25 @@ def __setattr__(self, key, value):
         super(_ResourceElement, self).__setattr__(key, value)
 
     def refresh(self):
-        """Make a REST call to refresh the JSON object stored within the object
-        Returns:
-            _ResourceElement: The provided object instance is returned with updated value
+        """Refresh the JSON representation of the resource for the latest status
```

Comment:
The JSON is an implementation detail, but it's also a necessary bit of information for user to access the properties.  When user print a resource element, this is what they see:
```
{
  'rest_self': 'https://blahblah',
  'json_rep': { 'resourceType': 'blah', 'name': 'blah', ... }
}
```

The JSON representation is exposed as the `json_rep` property.  They can access the properties via dot notation.  (i.e. element.resourceType).  I'd like to find a better description to explain this.